src/schema/rules/tabular_data/modality_agnostic.yaml

```diff
@@ -15,6 +15,9 @@ Participants:
     handedness: recommended
     strain: recommended
     strain_rrid: recommended
+    birthdate: recommended
+    age_category: recommended
+    age_type: optional
```

Comment:
```suggestion
    age_reference: recommended
```

Reply:
NOTE: the numeric age ALWAYS needs to have a reference start point (right now always assumes default being 'birth/start of neonate stage'). This should be made explicit in future through age_reference being required if numeric age is provided. Alternatively to numeric age the birth date of a subject can be provided for animal subjects (only for human subject if anonymized). 

Q1: how to anonymize human subject birth date and measurement dates?
Q2: how to provide longitudinal age information (with one age reference)